resources/js/src/app/mixins/url.js

```diff
@@ -277,6 +277,27 @@ export default {
             return selectedFilters;
         },
 
+        /**
+         * @param category
+         * @returns {boolean}
+         */
+        isCategorySelected(category) {
+            let selectedFilters = this.getSelectedFilters(),
+                splitedSelectedCategories = [];
+
+            for (let i = 0; i < selectedFilters.length; i++) {
+                if (selectedFilters[i].id !== this.facet.id) {
+                    continue;
+                }
+
+                splitedSelectedCategories = selectedFilters[i].name.split('>');
```

Comment:
Wouldn't this cause a conflict when multiple categories have the same name? E.g.

* Men > Shoes
* Women > Shoes

Reply:
Is this still an issue? If not, please resolve the conversation.